src/tribler/ui/public/locales/ko_KR.json

```
@@ -0,0 +1,165 @@
+    "CreateTorrent": "파일에서 토렌트 만들기",
+    "DefaultDownloadSettings": "기본 내려받기 설정",
+    "SaveFilesTo": "파일을 저장할 위치 :",
+    "AlwaysAsk": "항상 내려받기 설정 확인",
+    "DownloadAnon": "프록시를 사용하여 익명으로 다운로드",
+    "SeedAnon": "프록시를 사용하여 암호화된 익명 시딩",
+    "TorrentWatchFolder": "토렌트 폴더 보기",
+    "WatchFolder": "폴더 보기",
+    "FamilyFilter": "가족 필터",
+    "EnableFamilyFilter": "가족 필터 사용",
+    "HideTags": "콘텐츠 항목의 태그 숨기기",
+    "Save": "저장",
+    "Tags": "태그",
+    "ProxySettings": "토렌트 프록시 설정",
+    "Type": "형식",
+    "Server": "서버",
+    "Port": "포트",
+    "Username": "사용자명",
+    "BittorrentFeatures": "비트토렌트 기능",
+    "EnableUTP": "활성화된 대역폭 관리(uTP)",
+    "MaxConnections": "내려받기당 최대 연결 수",
+    "ZeroIsUnlimited": "0 = 무제한",
+    "Password": "비밀번호",
+    "UploadRate": "올리기 속도 제한",
+    "DownloadRate": "내려받기 속도 제한",
+    "RateLimitNote": "해당 설정은 익명 및 일반 다운로드에 적용됩니다.",
+    "RateUnit": "KB/s   (0 = 무제한)",
+    "SeedRatio": "업/다운 비율이 다음보다 클 때까지 시드",
+    "SeedForever": "무제한 시드",
+    "SeedTime": "시드 시간(분)",
+    "NoSeeding": "시드 없음",
+    "SeedingNote": "해당 설정은 기존 다운로드에도 적용됩니다.",
+    "MinHops": "고속\n최소 익명성",
+    "MaxHops": "저속\n높은 익명성",
+    "EnableDevMode": "개발자 모드 사용",
+    "EnableStats": "네트워크 통계 사용(다시 시작해야 함)",
+    "LogDir": "로그 디렉터리",
+    "Name": "이름",
+    "Size": "크기",
+    "Created": "생성",
+    "Status": "상태",
+    "Seeds": "시드",
+    "Peers": "피어",
+    "SpeedDown": "감속",
+    "SpeedUp": "가속",
+    "Ratio": "비율",
+    "Anonymous": "익명성?",
+    "Hops": "홉",
+    "ETA": "ETA",
+    "AddedOn": "추가됨",
+    "ForceRecheck": "강제 다시 확인",
+    "ExportTorrent": "토렌트 파일 내보내기",
+    "MoveStorage": "파일 저장소 이동",
+    "ChangeAnonymity": "익명성 변경",
+    "ZeroHops": "익명성 없음",
+    "OneHop": "원홉",
+    "TwoHops": "투홉",
+    "ThreeHops": "트리홉",
+    "Details": "자세히 보기",
+    "Progress": "진행",
+    "Filesize": "파일 크기",
+    "Health": "상태",
+    "Destination": "대상",
+    "Availability": "가용성",
+    "Files": "파일",
+    "Trackers": "트래커",
+    "PeerIpPort": "피어(IP/포트)",
+    "Flags": "플래그",
+    "Client": "클라이언트",
+    "SeedersLeechers": "{{seeders, number}} 시더, {{leechers, number}} 리더",
+    "NoResults": "검색 결과가 없습니다.",
+    "GotoFirst": "첫 페이지로 이동",
+    "GotoPrev": "이전 페이지로 이동",
+    "GotoNext": "다음 페이지로 이동",
+    "GotoLast": "마지막 페이지로 이동",
+    "DownloadTorrent": "토렌트 내려받기",
+    "DownloadExists": "참고 : 이 토렌트는 내려받기에 이미 존재합니다.",
+    "Download": "내려받기",
+    "Cancel": "취소",
+    "LoadingTorrent": "토렌트 파일 {{method, string}}을(를) 불러오는 중...",
+    "anonymously": "무기명",
+    "directly": "직접",
+    "MagnetDialogInputLabel": "아래 영역에 URL/마그넷 링크를 입력 :",
+    "MagnetDialogHeader": "URL/마그넷 링크에서 토렌트 추가",
+    "MagnetDialogError": "URL/마그넷 링크를 처리할 수 없음",
+    "TrackerDialogInputLabel": "아래 영역에 트래커 URL을 입력 :",
+    "TrackerDialogHeader": "트래커 URL에서 토렌트 추가",
+    "Add": "추가",
+    "FilterByName": "이름별 필터링",
+    "WithSelected": "선택한 항목 :",
+    "RemoveDownload": "내려받기 제거",
+    "RemoveDownloadData": "내려받기 제거 + 데이터",
+    "RemoveDownloadConfirm": "선택한 {{다운로드, 번호}}개의 다운로드를 삭제하시겠습니까?",
```

```suggestion
    "RemoveDownloadConfirm": "선택한 {{downloads, number}}개의 다운로드를 삭제하시겠습니까?",
```

These names need to remain in English.